content/docs/getting_started/visual_story/create_bookend@ko.md

```diff
@@ -2,33 +2,33 @@
 $title: 북엔드 만들기
 ---
 
-자, 이제 모든 페이지를 추가했습니다. 이번에는 스토리의 마지막 화면인 북엔드(bookend)를 살펴보겠습니다. 마지막 화면은 스토리를 마무리하면서 SNS 공유 또는 연관된 스토리에 대한 링크를 제공합니다. 독자들이 여러분의 스토리를 공유하거나 다른 컨텐츠로 자연스럽게 이동할 수 있습니다.
+모든 페이지를 추가했으므로 이제 스토리의 마지막 화면인 '북엔드'에 대해 살펴보겠습니다.  마지막 화면에서는 스토리를 마무리하고, 사용자가 스토리를 공유하거나 게시자 사이트의 다른 콘텐츠를 자세히 살펴볼 수 있도록 소셜 공유 기능 또는 관련 링크를 제공합니다.
 
-북엔드 화면의 정보는 `<amp-story-bookend>` 컴포넌트에 명시된 JSON 파일로부터 옵니다. 이번 자습서를 위해 JSON 파일([bookend.json](https://github.com/ampproject/docs/blob/master/tutorial_source/amp-pets-story/bookend.json))을 미리 만들어 두었습니다.
+북엔드 화면의 정보는 `<amp-story-bookend>` 태그에 지정된 JSON 파일에서 가져옵니다. 이 가이드에서는 북엔드 데이터가 포함된 JSON 파일([bookend.json](https://github.com/ampproject/docs/blob/master/tutorial_source/amp-pets-story/bookend.json))이 이미 만들어져 있습니다.
 
-`<amp-story-bookend>` 태그는 `<amp-story>`안에서 가장 마지막에 두어야 할 태그입니다. 따라서 `<amp-story-bookend></amp-story-bookend>`를 `</amp-story>`태그 **앞에 넣습니다**. `amp-story-bookend` 태그의 `src` 속성은 `bookend.json`으로 설정하고 `layout="nodisplay"` 속성을 추가합니다:
+`<amp-story-bookend>` 태그는 `amp-story`의 마지막 태그여야 합니다. 따라서 종료 `</amp-story>` 태그 바로 앞에 `<amp-story-bookend></amp-story-bookend>`를 **추가**합니다.  `amp-story-bookend` 태그에서 `src` 속성을 `bookend.json` 파일로 지정하고 `layout="nodisplay"`를 설정합니다.
 
 ```html hl_lines="2"
   </amp-story-page>
   <amp-story-bookend src="bookend.json" layout="nodisplay"></amp-story-bookend>
 </amp-story>
 ```
 
-웹브라우저에서 새로고침을 하고 마지막 화면까지 오면 다음과 같은 화면이 보일 것입니다:
+브라우저를 새로 고치고 마지막 화면으로 이동하면 북엔드가 다음과 같이 표시됩니다.
 
-{{ image('/static/img/docs/tutorials/amp_story/bookend_full.gif', 398, 709, align='center third', alt='Bookend' ) }}
+{{ image('/static/img/docs/tutorials/amp_story/bookend_full.gif', 398, 709, align='center third', alt='북엔드' ) }}
 
-JSON 파일을 살펴보겠습니다. 텍스트 에디터에서 [bookend.json](https://github.com/ampproject/docs/blob/master/tutorial_source/amp-pets-story/bookend.json) 파일을 열어보십시오.
+JSON 파일에 대해 살펴보겠습니다.  텍스트 편집기에서 [bookend.json](https://github.com/ampproject/docs/blob/master/tutorial_source/amp-pets-story/bookend.json) 파일을 엽니다.
 
-북엔드 데이터 파일에는 `bookendVersion`을 반드시 입력해야 하는데 우리는 `v1.0`을 쓰겠습니다:
+모든 북엔드 화면에는 `bookendVersion`(이 가이드의 경우 `v1.0`)이 필요합니다.
 
 ```json
 "bookendVersion": "v1.0",
 ```
 
-소셜 공유 버튼은 독자들이 여러분의 콘텐츠를 트위터와 페이스북 그리고 핀터레스트 등의 소셜 네트워크 플랫폼에 공유할 수 있도록 합니다. 소셜 네트워크의 목록은 shareProviders 객체에 넣습니다. 소셜 네트워크의 [type name](/ko/docs/reference/components/amp-social-share.html#pre-configured-providers)은 여기에서 확인하십시오.
+리더는 소셜 공유 버튼을 사용하여 Twitter, Facebook, Pinterest 등 소셜 플랫폼을 통해 콘텐츠를 공유할 수 있습니다. shareProviders 개체에서 소셜 공유 제공업체를 지정하고, 각 소셜 플랫폼에 대한 [유형 이름](/ko/docs/reference/components/amp-social-share.html#pre-configured-providers)을 포함하는 배열을 만듭니다.
```

Comment:
- English: "type names"
- Current: "유형 이름"
- Proposed: "타입명" ('타입' as a transliteration of 'type' would be better, it can be easier to understand the table in 'providers' page.)